sync/sync.go

```diff
@@ -317,7 +374,9 @@ func (s *Synchronizer) revertHead(forkBlock *core.Block) {
 	}
 
 	s.log.Infow("Reorg detected", "localHead", localHead, "forkHead", forkBlock.Hash)
-
+	if err != nil {
+		s.log.Warnw("Failed getting reverse state-diff, err: ", err)
+	}
```

Comment:
is this error check relevant? if so can we put it to if branch above i.e.:
```
if err == nil {
   ...
} else {
    s.log.Warnw(...)
}
```

Reply:
It's no longer relevant, sorry about that